apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -227,15 +255,15 @@ export default class TutorialExplorer extends React.Component {
    *
    * Whether en or non-en user, this filters as though the user is of "en-US" locale.
    */
-  filterTutorialSet(filters, sortBy, orgName) {
+  filterTutorialSet(filters, sortBy, orgName, searchTerm) {
     const grade = filters.grade[0];
-
     const filterProps = {
       filters: filters,
       hideFilters: this.props.hideFilters,
       locale: 'en-US',
       orgName: orgName,
-      sortByFieldName: this.getSortByFieldName(sortBy, grade)
+      sortByFieldName: this.getSortByFieldName(sortBy, grade),
+      searchTerm: searchTerm
```

Comment:
General comment for this review -- could we treat the `searchTerm` as one of the filters (instead of passing it separately from the existing set of filters everywhere)?  Or is `searchTerm` different from the other filters in some way?

Reply:
I treated `searchTerm` differently than the other filters because it's a string rather than on/off as indicated by whether the entry was checked or not. It's more like, `orgName` in that we're matching to a specific key which is tracked separately as it's own piece of state so I followed that pattern.